src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkConnectorConfig.java

```diff
@@ -132,7 +133,10 @@ protected static ConfigDef baseConfigDef() {
                   group, ++order, Width.LONG, "Topics for 'Ignore Key' mode")
           .define(TOPIC_SCHEMA_IGNORE_CONFIG, Type.LIST, "", Importance.LOW,
                   "List of topics for which ``" + SCHEMA_IGNORE_CONFIG + "`` should be ``true``.",
-                  group, ++order, Width.LONG, "Topics for 'Ignore Schema' mode");
+                  group, ++order, Width.LONG, "Topics for 'Ignore Schema' mode")
+          .define(BEHAVIOR_ON_NULL_VALUES_CONFIG, Type.STRING, "ignore", ConfigDef.ValidString.in("ignore", "delete", "fail"), Importance.LOW,
```

Comment:
You could get the default string from Enum instead of hardcoding. The Validator list can also be constructed in enum and used here.

Reply:
Ahhh, nifty! Will definitely change that.